### ISSUE TITLE
“Radio Group Example Using aria-activedescendant” does not work in iOS/VoiceOver

### ISSUE THREAD

**Issue by yatil** [w3c/aria-practices#2475]
[This demo](https://www.w3.org/WAI/ARIA/apg/example-index/radio/radio-activedescendant.html) seems to not work in iOS/VoiceOver. Radio buttons can be selected but not pressed using the space key or by moving the selection with the left/right arrow keys. Developers implement these code examples (yes, despite the warning) instead of just using the proper HTML which gives them everything that this demo does for free, and more.

Please at least provide examples that work.

**Comment by JAWS-test** [comment]
This is probably a bug of iOS/VoiceOver. The example itself works correctly. Please file a ticket at iOS/VoiceOver.

**Comment by accdc** [comment]
Hi,
As requested I have done some extensive testing with an iPhone running iOS16 with an external bluetooth keyboard.

As I mentioned during the meeting, it is impossible for the arrow keys to control the selection of a radio button group on iOS in the same manner as a Windows machine plus keyboard.

This is actually not an issue with the ARIA radio button group example, but rather is simply the way that iOS handles external keyboards.

For example, you can reproduce the same behavior here using native radio buttons.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio#try_it

It is impossible here to to change the checked state of any radio button with the arrow keys on native radios as well.

Voiceover will enter edit mode when you double tap on a text input field, and then you can use the arrow keys as expected to move the cursor which is similar to Windows, however that only works like this on edit fields and not on any other interactive control type such as radios, checkboxes, sliders, trees, listboxes, grids, tabs, nor anything else.
